source/Irrlicht/CMakeLists.txt

```diff
@@ -292,6 +292,11 @@ add_library(IrrlichtServer STATIC
 	$<TARGET_OBJECTS:zlib>
 )
 
+if (NOT MSVC)
+	target_compile_options(Irrlicht PRIVATE -fexceptions)
```

Comment:
why enable exceptions?

Reply:
check other mails too, I explained it in the issue